reversion/models.py

```diff
@@ -256,6 +257,8 @@ class AuditLog(models.Model):
     created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True, db_index=True)
     versions = models.ManyToManyField(Version, verbose_name=_('versions'))
     comment = models.TextField(verbose_name=_('comment'), blank=True, help_text=_('A text comment on this revision.'))
+    priority = models.PositiveIntegerField(verbose_name=_('priority'), null=True, blank=True)
```

Comment:
proč tohle nemá ChoicesNumEnum? chceš ať si to používá každý jak chce a ať může mít i jiné priority než LOW, MEDIUM a HIGH?